app/uk/gov/hmrc/partnershipidentificationfrontend/service/RegistrationOrchestrationService.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.partnershipidentificationfrontend.service
 
 import uk.gov.hmrc.http.{HeaderCarrier, InternalServerException}
 import uk.gov.hmrc.partnershipidentificationfrontend.connectors.RegistrationConnector
-import uk.gov.hmrc.partnershipidentificationfrontend.models.PartnershipType.{GeneralPartnership, PartnershipType, ScottishPartnership}
+import uk.gov.hmrc.partnershipidentificationfrontend.models.PartnershipType.{GeneralPartnership, LimitedLiabilityPartnership, LimitedPartnership, PartnershipType, ScottishLimitedPartnership, ScottishPartnership}
```

Comment:
As you are using all of the types defined in PartnershipType you could use the shorthand "import uk.gov.hmrc.partnershipidentificationfrontend.models.PartnershipType._"